src/shared/pages/welcomePage/localization.js

```diff
@@ -51,10 +68,8 @@ export function replaceDataLocale(id, href, platform = "") {
 export function applyLocalization() {
   const elements = document.querySelectorAll("[data-locale]");
   const hrefMapping = {
-    welcomePageManageShortcutsChromium: "chrome://extensions/shortcuts",
-    welcomePageManageShortcutsFirefox: "addons://shortcuts/shortcuts",
     welcomePageErrorChromium: "https://www.mozilla.org/firefox/new/",
-    privacyNoticeLink: "",
+    welcomePageTelemetryCheckbox: "https://example.com/",
```

Comment:
Add a TODO comment and/or file an issue to make sure that this example.com link gets replaced with the correct URL before shipping.